source/examples/fit-peaks.rst

```diff
@@ -0,0 +1,70 @@
+Fit and Analyze Peaks
+*********************
+
+Problem
+=======
+
+Scan a region where a peak is expected. Fit a nonlinear model to the
+measurements and characterize the peak center, full-width half-max, etc.
+
+Approach
+========
+
+Use the bluesky ``PeakStats`` callback. Here's how it works:
+
+1. Like ``LiveTable`` and ``LivePlot``, it receives a live stream of data.
+2. It silent accumulates this data until a run completes.
+3. It performs a nonlinear best fit, and it makes the results available through
+   attributes that can be used in plots, computations, or future scans.
```

Comment:
consider including the model(s) that peakstats uses (gaussian, gaussian+linear, etc...)